src/prioritize-features.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+""" Automatically prioritize features of one label above others
+
+Problem: If you use labels to track different categories of features and you want to prioritize all
+features of one label above others (treat it as the focus), you have to manually comb through the
+backlog and move the features up (increase their rank) for each one. If features in that label have
+different priority-field settings, you don't want to move them all to the top; you just want to move
+them above other features of the same priority field setting. What a pain!
+
+This script attempts to automate that for you.
+
+This script accepts one argument: a label. All of the features in that label will be prioritized up
```

Comment:
If we ignore the token (which is an 'admin' thing), my understanding is that it also requires a project (as per the click definition of the command). However as per my other comment, it seems possible for changes to impact non-related project.